front/app/containers/Admin/projects/project/projectHeader/ReviewFlow/index.tsx

```diff
@@ -17,29 +17,18 @@ import ReviewRequestButton from './ReviewRequestButton';
 
 const ReviewFlow = ({ project }: { project: IProjectData }) => {
   const isProjectReviewEnabled = useFeatureFlag({ name: 'project_review' });
-
   const { formatMessage } = useIntl();
-
   const { data: projectReview, isLoading: isProjectReviewLoading } =
     useProjectReview(project.id);
-
   const { mutate: updateProject, isLoading: isUpdatingProjectLoading } =
     useUpdateProject();
-
   const {
     mutate: approveProjectReview,
     isLoading: isApprovingProjectReviewLoading,
   } = useApproveProjectReview();
-
   const canPublish = usePermission({
     item: project,
     action: 'publish',
-    context: projectReview?.data.attributes.state === 'approved',
```

Comment:
This is now checked for in the place we define the "publish project" permission.